samigo/samigo-app/src/webapp/jsf/author/item/imageMapQuestion.jsp

```diff
@@ -205,7 +205,12 @@
         </div>
         <h:outputLabel value="#{authorMessages.image_map_alt}" styleClass="col-md-2 form-control-label"/>
          <div class="col-md-10">
-            <h:inputText id="imageMapAltText" value="#{itemauthor.currentItem.imageMapAltText}" styleClass="form-control" maxlength="254"/>
+            <sakai-text-input inputId="itemForm:imageMapAltText" 
+                value='<h:outputText value="#{itemauthor.currentItem.imageMapAltText}" escape="false" />'
+                maxLength="254"
+                errorMessage='<h:outputText value="#{authorMessages.image_map_alt_length_warning}" escape="false" />' 
+                styleClass="form-control"
```

Comment:
You shouldn't use camel case in attribute names. Some browsers break, or used to. Just use hyphenated names and map them in your component with the attribute object key.